src/d3a/d3a_core/sim_results/area_market_stock_stats.py

```diff
@@ -36,33 +35,28 @@ def update(self, area):
             self.state[area.name][last_past_market.time_slot] = {}
 
         for bid in last_past_market.bid_history:
-            tool_tip = f"{bid.buyer_origin} " \
```

Comment:
The changes you did to this file are not needed for this story, but to sync the information on the bids_offers_trades graph on the UI and the backend, right?

Reply:
Also the changes in export.py Right?

Reply:
It is needed as part of this story too as info needed for `high_resoulution_energy_trades` were part of tool_tip which is essentially an string as well as not consistent for offer/bids and trades, so quite hard and also not good approach to extract info from an string when such info could easily be saved in dict and tool_tips are only needed for backend plots and could be generated on fly while exporting backend plots.

Reply:
yep, makes sense, Thanks!